_episodes/15-transferring-files.md

```diff
@@ -8,183 +8,107 @@ objectives:
-for new users, but we'll break it down here:
+##  Compressing files
 
-To transfer *to* another computer:
-```
-[local]$ scp /path/to/local/file.txt yourUsername@remote.computer.address:/path/on/remote/computer
-```
-{: .bash}
+Let's look at the file we just downloaded:
 
-To download *from* another computer:
 ```
-[local]$ scp yourUsername@remote.computer.address:/path/on/remote/computer/file.txt /path/to/local/
+[remote]$ ls
+bash-lesson.tar.gz
 ```
-{: .bash}
-
-Note that we can simplify doing this by shortening our paths. On the remote computer, everything
-after the `:` is relative to our home directory. We can simply just add a `:` and leave it at that
-if we don't care where the file goes.
-
-```
-[local]$ scp local-file.txt yourUsername@remote.computer.address:
-```
-{: .bash}
 
-To recursively copy a directory, we just add the `-r` (recursive) flag:
-
-```
-[local]$ scp -r some-local-folder/ yourUsername@remote.computer.address:target-directory/
-```
-{: .bash}
-
-## Transferring files interactively with sftp
+You may recognize that this is a compressed file, or an *archive*, by the file extension `tar.gz`. 
+Sometimes it is convenient to compress files to make file transfers easier. 
+The larger the file, the longer it will take to transfer. Moreover, we can compress a whole bunch of little files
+into one big file to make it easier on us (no one likes transferring 70000 little files)!
 
-`scp` is useful, but what if we don't know the exact location of what we want to transfer? Or
-perhaps we're simply not sure which files we want to transfer yet. `sftp` is an interactive way of
-downloading and uploading files. Let's connect to a cluster, using `sftp`- you'll notice it works
-the same way as SSH:
+Let's extract this file with the `tar` command:
 
 ```
-sftp yourUsername@remote.computer.address
-```
-{: .bash}
+[remote]$ tar -xf bash-lesson.tar.gz
+[remote]$ ls
+bash-lesson.tar.gz                           SRR307023_1.fastq  SRR307025_1.fastq  SRR307027_1.fastq  SRR307029_1.fastq
+dmel-all-r6.19.gtf                           SRR307023_2.fastq  SRR307025_2.fastq  SRR307027_2.fastq  SRR307029_2.fastq
+dmel_unique_protein_isoforms_fb_2016_01.tsv  SRR307024_1.fastq  SRR307026_1.fastq  SRR307028_1.fastq  SRR307030_1.fastq
+gene_association.fb                          SRR307024_2.fastq  SRR307026_2.fastq  SRR307028_2.fastq  SRR307030_2.fastq
+``` 
 
-This will start what appears to be a bash shell (though our prompt says `sftp>`). However we only
-have access to a limited number of commands. We can see which commands are available with `help`:
+To compress files, we can use the `tar` command again, but this time with the `-c` 
+flag to indicate we want to compress files. The syntax for compressing files is 
+`tar -czf archive-name.tar.gz file1 file2 ...`. Let's make another archive that contains
+only the `fasta` files:
 
 ```
-sftp> help
+[remote]$ tar -czf bash-lesson_fastq.tar.gz *.fastq
```

Comment:
Please restore the `{: .bash}` tag for syntax highlighting,
and separate out the output into its own block with a `{: .output}` tag.